### ISSUE TITLE
pingBeforeActivateConnection and authentication fails using Redis Sentinel

### ISSUE THREAD

**Issue by coolmeen** [mp911de/lettuce#448]
hi.
i am trying to set pingBeforeActivateConnection option using sentinel architecture, the problem is that when i have a master password defined on my master/slave nodes so basically it look like -

```
clientSentinelUri = RedisURI.Builder.sentinel("sentinel-ip.org",26379, MASTER_NAME)
                .withPassword("blablabla")
                .withTimeout(Long.valueOf(1), TimeUnit.SECONDS)
                .build();

        sentinelClient = RedisClient.create(clientSentinelUri);
        sentinelClient.setOptions(ClientOptions.builder()
                .pingBeforeActivateConnection(**true**).build());

       sentinelClient.connect().sync().ping();
```
 
but then an exception is thrown

```
Caused by: java.util.concurrent.ExecutionException: com.lambdaworks.redis.RedisCommandExecutionException: ERR unknown command 'AUTH'
```

meaning the **sentinel** is trying to use the password instead of the **master**
when i set to false it works great-

` .pingBeforeActivateConnection(false).build());`

so basically i am stuck, i can create some dynamic clients on every connection but that is not what i hope to get.
also, i don't really need to ping the sentinel just the master/slaves.

**Comment by mp911de** [comment]
Hi, thanks for the bug report. Actually, that's a known bug and you're the one who filed the ticket first. 

Please disable `pingBeforeActivateConnection` for the time being. `pingBeforeActivateConnection ` was only needed at the time lettuce didn't had SSL support and client connections used a SSL tunnel. Sometimes, the SSL connection broke and the protocol state was unknown and that required a way to find out whether the tunnel/endpoint can correctly respond to a `PING`.

**Comment by mp911de** [comment]
That's fixed now.
